src/v/kafka/server/tests/quota_manager_test.cc

```diff
@@ -43,17 +43,6 @@ struct fixture {
     }
 };
 
-template<typename F>
```

Comment:
Now that everything is in a seastar thread, the fixture can be simplified:
```c++
struct fixture {
    kafka::quota_manager::client_quotas_t buckets_map;
    ss::sharded<cluster::client_quota::store> quota_store;
    ss::sharded<kafka::quota_manager> sqm;

    fixture() {
        quota_store.start().get();
        sqm.start(std::ref(buckets_map), std::ref(quota_store)).get();
        sqm.invoke_on_all(&kafka::quota_manager::start).get();
    }

    ~fixture() {
        sqm.stop().get();
        quota_store.stop().get();
    }
};
```

Reply:
Done